packages/openapi-generator/README.md

```diff
@@ -1,17 +1,18 @@
-## SAP Cloud SDK OpenAPI Client Generator (Beta)
+<a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
```

Comment:
BTW this could also be replaced, right? `<!-- sap-cloud-sdk-logo -->`

Reply:
yes good idea.